dash-spv-apple-bindings/build.sh

```diff
@@ -46,12 +46,19 @@ rustup target add x86_64-apple-ios
 rustup target add aarch64-apple-ios
 rustup target add aarch64-apple-ios-sim
 
-cargo lipo --release
-cargo build --target=x86_64-apple-darwin --release
-cargo build --target=aarch64-apple-darwin --release
-cargo build --target=x86_64-apple-ios --release
-cargo build --target=aarch64-apple-ios --release
-cargo build --target=aarch64-apple-ios-sim --release
+cargo lipo
+cargo build --target=x86_64-apple-darwin 
+cargo build --target=aarch64-apple-darwin
```

Comment:
I suppose you just want to make debug symbols accessible in XCode?

In this case you can control it [here](https://github.com/dashpay/dash-shared-core/blob/8a8c03caad1a39965cb4f426c2796a72e241992d/Cargo.toml#L18) by uncommenting `#debug = true` and/or playing around opt-level

Reply:
Ok, thanks, that'll be more convenient